.travis.yml

```diff
@@ -1,21 +0,0 @@
-sudo: false
-language: python
-
-python:
-  - 3.6
-  # TODO: re-enable these tests once the importlib_metadata version bug has
-  # been addressed
-  # - 3.7
-  # - 3.8
-  - pypy3
-
-env:
-  - TOX_ENV=core
-
-jobs:
-  include:
-    - python: 3.8
-      env: TOX_ENV=flake8
```

Comment:
I think we'll want to have at least one test [run the linter](https://github.com/imgix/imgix-python/blob/4f188b854f50a6cb942ef537588efcd79dfec6c2/CONTRIBUTING.md?plain=1) (flake8) just to match what CI was previously doing

Reply:
Resolved offline, looks like it's already running in each job